global.js

```diff
@@ -23,11 +31,15 @@ function isVisible (rect) {
 
 function getNextKeyCombination (index) {
   // use 1st half of alphabet for single letter, and 2nd half for double letter, so that no collisions can occur
-  if (index < 13) {
+  let halfIndex = Math.floor(alphabet.length / 2)
+  if (index < halfIndex) {
     return alphabet[index]
   } else {
+    // Subtract half-index from index such that lettering starts at first index for second letter, e.g.
+    // na nb nc nd ...
+    index -= halfIndex
     // two-letter combination
-    return alphabet[Math.floor(index / 13) + 12] + alphabet[index % 26]
+    return alphabet[Math.floor(index / alphabet.length) + halfIndex] + alphabet[index % alphabet.length]
```

Comment:
This seems different than the original logic - the original code goes `m`, `nn`, `no`, whereas this does `m`, `na`, `nb`. I don't actually remember at the moment what the original purpose of splitting the alphabet was though, so if you think the new way is fine then I'm good with it.

Reply:
I think the new way is fine indeed.